### PR TITLE
Newchange5

### PR DESCRIPTION
**Pull request by almozaai**
Now order hub is working and lineItems then I will progress to be the products persists to user

### CODE REVIEW

File: db.js [hunk 1]
```diff
@@ -43,6 +43,11 @@ const Product = conn.define('product', {
     validate: {
       isDecimal: true
     }
+  },
```
Comment (arthurs14) [inline db.js:46]: We have this added already.

File: db.js [hunk 2]
```diff
@@ -111,13 +116,13 @@ const syncAndSeed = async () => {
...
 
   const products = [
-    { name: 'Hammer', price: 25.99 },
-    { name: 'Nails', price: 9.5 },
-    { name: 'Paint', price: 4.5 },
-    { name: 'Chair Set', price: 157.47 },
-    { name: 'Shovel', price: 15.0 },
-    { name: 'Lawn Mower', price: 300.25 },
-    { name: 'Wrench', price: 13.99 }
+    { name: 'Hammer', stock: 200, price: 25.99 },
```
Comment (arthurs14) [inline db.js:119]: Added this too.

File: db.js [hunk 3]
```diff
@@ -111,13 +116,13 @@ const syncAndSeed = async () => {
...
+    { name: 'Nails', stock: 200, price: 9.5 },
```
Comment (arthurs14) [inline db.js:120]: 120 - 197 already added.

**Review by arthurs14 — COMMENTED** [review]
There are some changes that already exist on the dev branch of this repo. Please make sure to pull the most recent so that your master is up to date, keep your edits on the branch you were working on and make sure to create a new branch to add the changes that are not already on the main branch. Mostly the database has the changes you have made. Not sure if its going to break our code but just a precaution.

**Comment by arthurs14** [comment]
Also there are some conflits on the server/routes.js and src/redux/thunk.js above. Make sure to take a look at those before doing another pull request.

File: src/redux/thunk.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ import {
   keepSession,
   getUsers,
   createUsers,
+  deleteUser,
```
Comment (nikkiconsolacion) [inline src/redux/thunk.js:9]: please add 'updateUser' here as well. I think you accidentally took it out during reformatting

**Pull request closed without merging**